nixos/modules/hardware/video/nvidia.nix

```diff
@@ -186,6 +195,18 @@ in
       ++ optional (nvidia_x11.persistenced != null && config.virtualisation.docker.enableNvidia)
         "L+ /run/nvidia-docker/extras/bin/nvidia-persistenced - - - - ${nvidia_x11.persistenced}/origBin/nvidia-persistenced";
 
+    systemd.services."nvidia-persistenced" = mkIf nvidiaPersistencedEnabled {
+      description = "NVIDIA Persistence Daemon";
+      wantedBy = [ "multi-user.target" ];
+      serviceConfig = {
+        Type = "forking";
+        Restart = "always";
+        PIDFile = "/var/run/nvidia-persistenced/nvidia-persistenced.pid";
+        ExecStart = "${pkgs.linuxPackages.nvidia_x11.persistenced}/bin/nvidia-persistenced --verbose";
```

Comment:
Change `pkgs.linuxPackages.nvidia_x11.persistenced` to `nvidia_x11.persistenced`.

Reply:
ok.